src/mcrypt/ctr.c

```diff
@@ -93,14 +93,20 @@ GF_Err _mcrypt_set_state(void *_buf, void *IV, int size)
 
 GF_Err _mcrypt_get_state(void *buf, void *IV, int *size)
 {
-	if (*size < ((CTR_BUFFER* )buf)->blocksize + 1) {
-		*size = ((CTR_BUFFER* )buf)->blocksize + 1;
+	if (*size < ((CTR_BUFFER* )buf)->blocksize) { 
+		*size = ((CTR_BUFFER* )buf)->blocksize;
 		return GF_BAD_PARAM;
 	}
-	*size = ((CTR_BUFFER* )buf)->blocksize + 1;
```

Comment:
Can you point me to the location in the CENC spec where the it should be blocksize+1 for the current counter after a subsample encrypt (with remaining counter)?  I didn't see anything that would suggest we needed to +1 byte but we may have missed something.

Our CENC decryption routines don't skip any bytes.  E.g. if 8 bytes of counter is left, we consume the 8 bytes of the counter.